tests/multichan_throughput_test.c

```diff
@@ -74,7 +74,7 @@ throughput_test (int converter, int channels, long best_throughput)
 		exit (1) ;
 		} ;
 
-	throughput = lrint (floor (total_frames / duration)) ;
+	throughput = (long) floor (total_frames / duration) ;
```

Comment:
Not the same.

Reply:
See above reasoning https://github.com/erikd/libsamplerate/pull/69#discussion_r306711585